src/components/Header.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 
-import { HeaderProps } from '../types';
+import { FirstHeader } from '../styles/header';
```

Comment:
With the new package regading the relative path , shouldn't it be '/styles/header' isntead of '../styles/header' ?